roles/artifacts/tasks/edpm.yml

```diff
@@ -81,6 +81,9 @@
           sudo ip ro ls >> /tmp/{{ host_ip }}/network.txt
           sudo rpm -qa > /tmp/{{ host_ip }}/rpm_qa.txt
           sudo podman images > /tmp/{{ host_ip }}/podman_images.txt
+          mkdir -p /tmp/{{ host_ip }}/service_logs
+          ls /etc/systemd/system/edpm_* | xargs -I {} echo {} | sed s'#/etc/systemd/system/##'g | awk -F '.' '{print $1}' | xargs -I {}  sudo bash -c 'journalctl -u {} > /tmp/{{ host_ip }}/service_logs/{}.log'
+          ls /etc/systemd/system/*virt* | xargs -I {} echo {} | sed s'#/etc/systemd/system/##'g | awk -F '.' '{print $1}' | xargs -I {}  sudo bash -c 'journalctl -u {} > /tmp/service_logs/{}.log'
```

Comment:
i can compact this to something like this
```
mkdir -p /tmp/{{ host_ip }}/service_logs
systemctl list-units | awk '/virt|edpm/ {print $1}' | grep -v sys | xargs -I {} sudo bash -c 'journalctl -u {} > /tmp/service_logs/{}.log'
```

Reply:
maybe listing the units would indeed be better?
Also, I think we should push everything in `{{ host_ip }}` - guess you missed that level in the second line (as well as your "reduction proposal") - else it won't be copied on the controller-0.

Reply:
oh that was a typo

they should have all been in /tmp/{{ host_ip }}/service_logs/{}.log'

ill fix that when i respin

Reply:
ya thats the problem i just need to change
```
 ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no \
-i {{ ssh_key_file }} {{ ssh_user }}@{{ host_ip }} <<EOF
```

to 

```
 ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no \
-i {{ ssh_key_file }} {{ ssh_user }}@{{ host_ip }} <<'EOF'
```